src/__tests__/TestDeduplicator.ts

```diff
@@ -3,6 +3,10 @@ import { deepEquals } from "../utils";
 import { AvoInspector } from "../AvoInspector";
 import { defaultOptions } from "./constants";
 
+jest
+  .useFakeTimers('modern')
+  .setSystemTime(new Date('2020-01-01'));
+
```

Comment:
💬 There is time-sensitive logic inside the library, which causes test failures if tests are running longer than expected, so this was added to remove the dependency on the running time of tests.